irods/webdav.yml

```diff
@@ -437,10 +437,9 @@
       when: purgemanResp.stat.exists
 
     - name: check purgeman service account
-      shell: grep purgeman -q /etc/passwd
+      shell: id purgeman
       register: purgemanAccount
-      ignore_errors: true
-      changed_when: false
```

Comment:
`changed_when: false` is still needed, otherwise ansible erroneously records that this task changed something.

Reply:
Added.